src/ui/components/results/directanswercomponent.js

```diff
@@ -210,7 +229,8 @@ export default class DirectAnswerComponent extends Component {
   }
 
   eventOptions (data) {
-    if (!data || Object.keys(data).length === 0) {
+    if (!data || Object.keys(data).length === 0 ||
+      (Object.keys(data).length === 1 && Object.keys(data)[0] === 'searchState')) {
```

Comment:
do we need this extra guarding?

Reply:
yeah, if there's no direct answer found in live api response after a complete search, directAnswer will have a SEARCH_COMPLETE state but no actual directAnswers `data` field, which `eventOptions` would throw an error trying to access `data.relatedItem.verticalConfigId`

Reply:
could we either guard this by the existence of relattedItem.verticalConfigId then, or use optional chaining?

Reply:
hmm optional chaining would work. Updated!